### PR TITLE
Do not report "blocked access" to Rollbar

### PR DESCRIPTION
**Pull request by MDrakos**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1609" title="DX-1609" target="_blank">StoryDX-1609</a>  Don't report "blocked access" to rollbar
  </td></table>
  <br />

### CODE REVIEW

File: internal/locale/locales/en-us.yaml [hunk 1]
```diff
@@ -2043,3 +2043,5 @@ runtime_limit_reached_action:
   other: Upgrade
 pjfile_deprecation_entry:
   other: " - '[ACTIONABLE]{{.V0}}[/RESET]' located at byte [ACTIONABLE]{{.V1}}[/RESET]"
+err_country_blocked:
+  other: Your country is blocked
```
Comment (mitchell-as) [inline internal/locale/locales/en-us.yaml:2047]: Oof. That's a bit harsh. We might want to soften the language a bit, but I'm not sure how.

File: internal/rollbar/rollbar.go [hunk 2]
```diff
@@ -23,9 +23,23 @@ type config interface {
 	Closed() bool
 }
 
-var currentCfg config
+type doNotReport []string
 
-var reportingDisabled bool
+func (d *doNotReport) Add(msg string) {
```
Comment (mitchell-as) [inline internal/rollbar/rollbar.go:28]: If you accidentally pass `locale.T("does_not_exist")`, does it return an empty string? If so, would that end up blocking all rollbar reports? If so, we might want to ignore empty strings here and possibly report them to rollbar so we can fix.

**Review by mitchell-as — CHANGES_REQUESTED** [review]
Nice solution :) I wrote some thoughts to consider.

File: internal/locale/locales/en-us.yaml (hunk 1 above)
Comment (MDrakos) [reply internal/locale/locales/en-us.yaml:2047]: Agreed it is a bit rough, but it is from the ACs of the story. @Naatan do we want to consider a more friendly error message?

File: internal/rollbar/rollbar.go (hunk 2 above)
Comment (MDrakos) [reply internal/rollbar/rollbar.go:28]: Thank you for catching this! It should be addressed now and I've added some tests to verify.

**Review by mitchell-as — DISMISSED** [review]

File: internal/locale/locales/en-us.yaml (hunk 1 above)
Comment (Naatan) [reply internal/locale/locales/en-us.yaml:2047]: @rawktron ^

Comment (rawktron) [reply internal/locale/locales/en-us.yaml:2047]: "This service is not available in your region."

**Review by mitchell-as — APPROVED** [review]

**Pull request merged**